vowpalwabbit/vw.sln

```diff
@@ -47,30 +41,16 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_testhelp", "..\cs\testhe
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "vw_common", "..\cs\common\vw_common.csproj", "{E621E022-C1F8-433F-905A-AB9A3DE072B7}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_leaktest", "..\cs\leaktest\cs_leaktest.csproj", "{492EA3A7-8A41-459E-BBC3-5A3FE9DDFC68}"
-EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_testcommon", "..\cs\testcommon\cs_testcommon.csproj", "{6A9CBEAB-427F-4D8D-9559-B76B42B0895C}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "cs_vld", "..\cs\cli\cs_vld.vcxproj", "{3D57A6AF-DE8C-40DC-ABDE-E4CE1B9C0D20}"
-EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "vw_dynamic", "vw_dynamic.vcxproj", "{1E205806-7F80-47DD-A38D-FC08083F3593}"
+Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "vw_core", "vw_core.vcxproj", "{1E205806-7F80-47DD-A38D-FC08083F3593}"
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "cs_console", "..\cs\cs_console\cs_console.csproj", "{01A85382-C3E9-480A-86BF-FAFE4AD107A7}"
 EndProject
-Project("{930C7802-8A8C-48F9-8165-68863BCCD9DD}") = "setup", "..\cs\setup\setup.wixproj", "{1F25D39D-71C3-4FDE-A58F-D3D5D4F6DFFB}"

```

Comment:
Any reason to keep the files for this project?

Reply:
Yes. While building the setup is not meaningful for most people building VW from source, it would be nice to have an installable VW package for those that just want to use the command-line form.

Reply:
got it.  In order to invoke it, would I need to open the setup project?

Reply:
We should create a separate solution for setup, at the least. I'll do that in a separate PR.

Reply:
Got it.  Thanks.